### PR TITLE
Dev to master for version v1.2.0

### PR DESCRIPTION
**Pull request by Husseinhj**
#### Changes:
 - Fix changing Chabok environment.
 #### Upgrade note:
- For change Chabok environments use `devMode` parameter in `init` method.
	`init(APP_ID, API_KEY, USERNAME, PASSWORD, DEVMODE)`
- Remove `appName` parameter in `initializeApp` method.
	`initializeApp(options)`
- The `setDevelopment` method is not available anymore. Use `devMode` parameter in `init` or `devMode` key in `initializeApp` method instead

### CODE REVIEW

**Comment by behrad** [comment]
is this ok @amir-yaghoobi ?

**Comment by behrad** [comment]
Shall we update React Bridge documentation afterwards @Husseinhj ?

**Comment by Husseinhj** [comment]
@behrad Yes should update documentation for v1.2.0 usage.

File: lib/modules/core/chabok.js [hunk 1]
```diff
@@ -11,19 +11,31 @@ export const playServicesAvailability = AdpNativeModule.playServicesAvailability
 
 export class AdpPushClient {
 
-    initializeApp = (appName, options, cbk) => {
-        AdpNativeModule.initializeApp(appName, options, response => {
-            cbk(response);
-        });
+    initializeApp = (options) => {
+        if (!options){
```
Comment (amir-yaghoobi) [inline lib/modules/core/chabok.js]: check options object keys

**Comment by amir-yaghoobi** [comment]
@behrad javascript code is valid, but personally, I didn't test it my self.

File: lib/modules/core/chabok.js (hunk 1 above)
Comment (behrad) [reply lib/modules/core/chabok.js]: You are reviewing after approval !? How is this approved then?

Comment (amir-yaghoobi) [reply lib/modules/core/chabok.js]: After my discussion with @Husseinhj we decided to hold the PR for extra changes and after that reviewing it.

Comment (behrad) [reply lib/modules/core/chabok.js]: 👍

**Comment by behrad** [comment]
Please let me know when this gets ready to be merged

**Pull request merged**